src/c++/perf_analyzer/data_loader.h

```diff
@@ -179,10 +179,16 @@ class DataLoader {
   cb::Error ValidateTensorShape(
       const std::vector<int64_t>& shape, const ModelTensor& model_tensor);
 
+  /// Helper function to validate the provided data's size
```

Comment:
Following the pattern above, we should probably add doxygen param comments here.
In general, we need to formalize on whether we do this for private methods or not.

Reply:
fixed